.github/workflows/readme-updates.yml

```
@@ -0,0 +1,18 @@
+name: README updates with the latest blog posts
+
+on:
+ schedule:
+   # Every hour at minute 0
+   - cron: '0 * * * *'
```

I think once a day should be enough (we pay for GitHub actions resources).

```suggestion
   - cron: '0 0 * * *'
```

---

I agree with your proposal and pushed the related commit. 👍 
Do you want to review the PR again or I merge it directly?

---

My approval still stands, :shipit:!